src/vehicle.cpp

```diff
@@ -5613,7 +5554,7 @@ void vehicle::update_time( const calendar &update_to )
 
     // Weather stuff, only for z-levels >= 0
     // TODO: Have it wash cars from blood?
-    if( funnels.empty() && solar_panels.empty() ) {
+    if( funnels.empty() ) {
```

Comment:
Don't drop the early escape though. This is one of the most expensive vehicle calculations.